src/continuation/continuation_clustering.jl

```diff
@@ -0,0 +1,165 @@
+It uses clustering across of features across a parameter range, potentially weighted
+by the distance in parameter space. Its input `mapper` must have
+a `GroupByClustering` as its grouping configuration.
+This is the original continuation method described in [^Gelbrecht2020].
+
+## Keyword Arguments
+- `info_extraction::Function` a function that takes as an input a vector of features
+  (corresponding to a cluster) and returns a description of the cluster.
+  By default, the centroid of the cluster is used.
+- `par_weight = 0` The distance matrix between features has a special extra weight that
+  is proportional to the distance `|p[i] - p[j]|` between the parameters used when
+  extracting features. This keyword argument is the weight coeficient that ponderates
+  the distance matrix. Notice that the range of parameters is normalized from 0 to 1
+  such that the largest distance in the parameter space is 1. The normalization is done
+  because the feature space is also (by default) normalized to 0-1.
+- `use_mmap = false` this parameter whether the feature distance matrix should be computed
+  in memory or on the disk using memory map. Should be used if a matrix with side
+  `length(prange)*samples_per_parameter` exceeds available memory.
+
+## Description
+
+The method first integrates and computes a set of statistics on the trajectories, for example
+the mean and standard deviation of the trajectory data points. Once all the featured statistics
+have been  computed, the algorithm computes the distance between each feature. A special weight
+term is added so that the distance between features with different parameters is increased.
+It helps to discriminate between attractors with very different parameters. At last, the
+distance matrix is clustered with the DBSCAN algorithm.
+
+[^Gelbrecht2021]:
+    Maximilian Gelbrecht et al 2021, Monte Carlo basin bifurcation analysis,
+    [New J. Phys.22 03303](http://dx.doi.org/10.1088/1367-2630/ab7a05)
+"""
+function ClusteringAcrossParametersContinuation(
+        mapper::AttractorsViaFeaturizing;
+        info_extraction = mean_across_features,
+        samples_per_parameter = 100,
+        par_weight = 0.0,
+        use_mmap = false,
+    )
+    if !(mapper.group_config isa GroupViaClustering)
+        throw(ArgumentError(
+            "This method needs `GroupViaClustering` as the grouping configuration."
+        ))
+    end
+    return ClusteringAcrossParametersContinuation(
+        mapper, info_extraction, samples_per_parameter, par_weight, use_mmap
+    )
+end
+
+function mean_across_features(fs)
+    means = zeros(length(first(fs)))
+    N = length(fs)
+    for f in fs
+        for i in eachindex(f)
+            means[i] += f[i]
+        end
+    end
+    return means ./ N
+end
+
+function basins_fractions_continuation(
+        continuation::ClusteringAcrossParametersContinuation, prange, pidx, ics;
+        show_progress = true, samples_per_parameter = 100
```

Comment:
`samples_per_parameter` is also defined here. What is the best place for the definition?

The algorithm was taking 100 instead of 1000 samples. It may be the source of the problem.

Reply:
`samples_per_parmater` is passed as a keword to the `basins_fractions_continuation` function. I'll update the samples now.